### DISCUSSION TITLE
Dos it require vj base?

### DISCUSSION THREAD

**Discussion by IvanVladimir04** [VJ-HLR-Developers/Half-Life-Resurgence#37] [Q&A] [answered]
i saw a discussssin saying that nav_generate works with vj base and when i tried they still not move in the skybox!!1!11!!1

**Comment by stants12** [comment, marked as answer]
try removing hl2.exe from I:\Steam\steamapps\common\GarrysMod

**Reply by DustyBuster64** [reply]
> try removing hl2.exe from I:\Steam\steamapps\common\GarrysMod

help, i can't turn on my pc anymore :(

**Reply by stants12** [reply]
have you tried unplugging it and plugging it back

**Reply by DrVrej** [reply]
Well as you can see from his profile picture, he is running on Windows 95, HLR is supported from Windows 98 only, not 95.
